docs-ui/components/panels.stories.js

```diff
@@ -71,6 +71,17 @@ export const _PanelTable = withInfo({
       <div>Panel Item</div>
       <div>Panel Item</div>
       <div>Panel Item</div>
+      <div>Panel Item</div>
+      <div>Panel Item</div>
+    </PanelTable>
+
+    <PanelTable
+      // eslint-disable-next-line react/jsx-key
```

Comment:
I think in case of string you don't need to add a key... so its fine to remove this lint disable

Reply:
![image](https://user-images.githubusercontent.com/29228205/106033944-968aa780-60d2-11eb-8620-e6295ffad80f.png)